### ISSUE TITLE
seemingly wrong condition preventing set attributes on iOS.

### ISSUE THREAD

**Issue by breakeren** [urbanairship/airship-mobile-framework-proxy#10]
Currently attempts to set attributes on iOS will fail, because setting non-empty attributes will be skipped.

add exclamation mark to AttributeOperation.swift:31 to fix it. I am no swift expert, but this solves it for me.
It would be nice with a fix from your side.

:

```
guard !attribute.isEmpty else {
    AirshipLogger.error("Invalid attribute operation: \(self)")
    return
}
```

I am using airship through react-native-airship v. 15.2.4.

**Comment by rlepinski** [comment]
yep... thanks for the report. Ill get some tests and a release out as soon as i can

**Comment by breakeren** [comment]
Thank you for fixing. How does this work then? Will @ua/react-native-airship get a new release with an updated reference to airship-mobile-framework-proxy version 2.0.8? And if so, what could the timeline be? Just need to know whether it is worth patching for me in my own project, or if I should wait.

**Comment by rlepinski** [comment]
Ill try to get a new react-native release out today

**Comment by rlepinski** [comment]
Fixed in 15.2.5
